index.bs

```diff
@@ -567,5 +619,5 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
 
   A special thanks to Mike West for his invaluable contributions to the initial
   version of this spec. Thanks to Brad Hill, Anne van Kesteren, Jonathan
-  Kingston, Mark Nottingham,  Sergey Shekyan , Dan Veditz, Eduardo Vela,
+  Kingston, Mark Nottingham, Sergey Shekyan, Dan Veditz, Eduardo Vela,
```

Comment:
Feel free to add yourself here!

Reply:
Good point 🙂 https://github.com/w3c/webappsec-subresource-integrity/pull/103/commits/5fa139f07a949620b097450a1aa2457674c69685